src/CategoryItem.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Created by Katarzyna_Bak on 27.03.2017.
+ */
+
+import React, { Component } from 'react';
+import './CategoryItem.css';
+
+
+function Subcategories(props) {
+    if (props.list.length === 0) {
+        return null;
+    }
+
+    const subcategoryItems = props.list.map((category) =>
+        <CategoryItem key={category.num.toString()}
+                      item={category}
+                      action={props.action}
+                      active={props.active}/>
+    );
+
+    return (
+        <ul>
+            {subcategoryItems}
+        </ul>
+    );
+}
+
+
+class CategoryItem extends Component {
+    // constructor(props) {
+    //     super(props);
+    // }
+
+    render() {
+
+        let acitveClass = '';
+        if (this.props.item === this.props.active && this.props.item.subcategories.length > 0) {
+            acitveClass = 'active';
+        }
+
+        return (
+            <li className={"category-list__item " + acitveClass} onClick={e => this.props.action(this.props.item) }>
```

Comment:
`e` in `onClick` can be ommited, if not used -> `() => this.props.action(this.props.item)`